docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -524,12 +524,6 @@ Rate control across all tablets being remote bootstrapped from or to this proces
 
 Default: `256MB`
 
-##### --yb_num_shards_per_tserver
```

Comment:
These are tserver flags too. So, we should document them here as well.

Reply:
Added to `yb-tserver` configuration options.